cmd/certificates/sign.go

```diff
@@ -5,43 +5,39 @@ package main
 
 import (
 	"crypto/x509"
+	"log"
```

Comment:
Use zap.Logger instead, mixing multiple logging approaches usually ends up looking bad on the console.